geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HScanExecutor.java

```diff
@@ -44,6 +45,9 @@
   @Override
   public RedisResponse executeCommand(Command command,
       ExecutionHandlerContext context) {
+
+    final UUID CLIENT_ID = context.getClientUUID();
+
     List<byte[]> commandElems = command.getProcessedCommand();
```

Comment:
This is my fault... but if we want to match native Redis, the `count` parameter should actually be a long... and only return the `ERR value is not an integer or out of range` message if it exceeds the maximum long capacity (9223372036854775807)... we'll have to change this in all the SCANs (if we decide to).

Reply:
I think the thought was that a cursor "unit" represents more than a single value in a redis hash (let say 100), and that if we have a max int*100 values in a single hash we would have a different point of failure.

Reply:
I made a story to look into this parameters for all the scans and document differences.  Going to leave this as an int for now.  Yell at me if you strongly disagree.

Reply:
I think it makes sense to leave it as an `int` given the cursor parameter will be an `int`  and max size of a hash is the same as an `int`. Thanks for discussing this with everyone!